zebra/zebra_vty.c

```diff
@@ -778,6 +834,7 @@ DEFPY(ip_route_address_interface,
 	  |(1-255)$distance                            \
 	  |vrf NAME                                    \
 	  |label WORD                                  \
+	  |table (1-4294967295)                        \
```

Comment:
This needs to be a separate defun so the table keyword does not show up unless namespaces are configured.

Reply:
I had done it for vrf node commands, but was thinking that it was ok for default vrf. so I updated it.